doc/whats-new.md

```diff
@@ -1,157 +1,289 @@
+
+The `/extract/` API is based on an underlying `extract` Python API, that
+can be used directly in scripts, with the same guarantees of portability
+across engine versions.
+
+There is now an `abort` WebAPI and an `abort` button in the WebUI and
+QGIS plugin, using which it is possible to cleanly kill a running
+calculation.
+
+We fixed a small issue: now when the WebUI is stopped all calculations
+started by it are automatically killed.
+
+The integration between the plugin and the web tools (in particular
+the Input Preparation Toolkit) has been improved as well; for the new
+features of the plugin please read its release notes.
+
+Bug fixes/additional checks
+------------------------------
+
+We fixed a minor bug with the .npz exports: the command
+`oq engine --run job.ini --exports npz` now works as expected.
+
+We raise a clear error when the parameter `risk_investigation_time`
+is missing in the `job_risk.ini` file.
+
+There is a better error message when the user wants to compute
+loss maps and the parameter `asset_loss_table` is missing.
+
+There is a clear warning when the engine produces no output, usually
+because `mean_hazard_curves` is set to `false`. Also, in that case, no
+output rows are incorrectly added to the database.
+
+There is a check on the path names used in the source model logic
+tree file: they must be relative paths, not absolute paths, to make it
+possible to port the input files across different machines and operating
+systems.
+
+There is a check on calculations starting from precomputed hazard results: if
+job.ini file contains a site model or logic tree files, the user is warned
+that they will not used for the subsequent calculation and should be removed.
+
+We improved the error message in the case of source models in format
+nrml/0.4 uncorrectly marked as nrml/0.5.
+
+The CSV rupture exporter now also exports the parameter
+`ses_per_logic_tree_path` in the header of the CSV file.
+
+A lot of work went into the .rst reports which are now more reliable than
+ever.
+
+oq commands
+-----------
+
+There is a new command `oq shell` that starts an IPython shell (or Python
+shell, if IPython is not available) in the engine virtual environment.
+This is very useful to explore interactively the results of a computation.
+
+There is a new command `oq abort <job_id>` that allows to kill
+properly running calculations.
+
+The command `oq engine
+--delete-calculation <job_id>` has been extended to abort running calculations
+before removing them from the database and the file system.
+
+The command `oq zip` has been extended and now works for all kind of
+calculations.
+
+The command `oq extract` has now a signature consistent with `oq export`
+and has grown the ability to connect to a remote engine server.
+
+The `oq dump` command has been extended so that it is possible to extract
+a single calculation, if so desired.
+
+IT
+---
+
+We have specialized packages for clusters. The packages meant for the controller
```

Comment:
..., also for Ubuntu.

RHEL was already there since couple of versions ago

Reply:
Done